services/ui/ws/display.cc

```diff
@@ -387,7 +387,7 @@ ServerWindow* Display::GetActiveRootWindow() {
 }
 
 bool Display::CanHaveActiveChildren(ServerWindow* window) const {
-  return window && activation_parents_.Contains(window);
+  return window && (activation_parents_.Contains(window) || window == root_.get());
```

Comment:
Added the comment.

In case of chromeos, this check is not needed, the window manager server window can have focus to its children. So we do not reach Display::root_ (as we do in Ozone/LinuxOS).

alternatively, we could also add something like:

 134 void WindowTree::DoOnEmbed(mojom::WindowTreePtr tree,
 135                            ServerWindow* root_window) {
 136   bool in_external_window_mode = !!root_window;
 137   if (in_external_window_mode) {
 138     DCHECK(!tree);
 139     CHECK_LE(1u, roots_.size());
 140 
 141     Display* display = GetDisplay(root_window);
 142     int64_t display_id =
 143         display ? display->GetId() : display::kInvalidDisplayId;
 144 
 145     ClientWindowId window_id;
 146     IsWindowKnown(root_window, &window_id);
 147 
 148     // In external window mode, every platform window is an activation parent.
 149     AddActivationParent(window_id);
 150+   AddActivationParent(ws:display's root server window);
 (...)